### PR TITLE
Implement Extended SDP Dissection

### PR DESCRIPTION
**Pull request by JeremyWhaling**
Add support for extended SDP dissection following the SAE J3400 RP document.

### CODE REVIEW

File: dissector/v2g.lua [hunk 1]
```diff
@@ -18,17 +18,33 @@ local SDP = {
...
     response_security = ProtoField.uint8("sdp.ResponseSecurity", "SDP Response Security", base.HEX),
     response_transport_proto = ProtoField.uint8("sdp.ResponseTransportProtocol", "SDP Response Transport Protocol", base.HEX),
     response_emsp_ids = ProtoField.string("sdp.ResponseEMSPIDs", "EMSP IDs"),
+    response_esdp_version = ProtoField.uint16("sdp.ESDPVersion", "ESDP Version", base.HEX),
+    response_esdp_reserved = ProtoField.uint16("sdp.ESDPReserved", "ESDP Reserved", base.HEX),
+    response_charge_handle_type = ProtoField.uint16("sdp.RequestChargeHandleType", "ESDP Charge Handle Type", base.HEX),
+    response_evse_id = ProtoField.string("sdp.ResponseEVSEID", "EVSE ID"),
+    response_evse_diagnostic_this_session = ProtoField.string("sdp.ResponseEVSEDiagnosticThisSession", "EVSE Diagnostic/Error Codes [this session]"),
+    response_evse_diagnostic_previous_session = ProtoField.string("sdp.ResponseEVSEDiagnosticPreviousSession", "EVSE Diagnostic/Error Codes [previous session]"),
+	response_esdp_tlv_evse_specific = ProtoField.string("RequestTLVEVSESpecific", "EVSE Specific"),
```
Comment (jhart-cpi) [inline dissector/v2g.lua:47]: Need to check your alignment with other surrounding code, this looks off.

**Comment by JeremyWhaling** [comment]
Ok, I should have fixed the spacing throughout. Please give it another once over and if good I'll squish the commits.

**Comment by JeremyWhaling** [comment]
After review of PCAPs at Testival, fixed some minor stuff and squished the commit, please review.

**Comment by jhart-cpi** [comment]
@JeremyWhaling can you attach a screenshot and a pcap with the additional data decoded?

**Comment by JeremyWhaling** [comment]
> @JeremyWhaling can you attach a screenshot and a pcap with the additional data decoded? This looks good otherwise and will be ready to merge.

I send the screenshots and pcap to Peter for you. While my example isn't super secret, I would prefer if it wasn't posted publicly since these are "real" vehicles/EVSE being used.

**Review by jhart-cpi — APPROVED** [review]

**Comment by jhart-cpi** [comment]
Closes #64

**Pull request merged**